### PR TITLE
PORT-7441 Add scheduled gitlab mode to incident mgt, code security, cloud cost and more

### PR DESCRIPTION
**Pull request by phalbert**
# Description

Integrations (1):
- PagerDuty
- ServiceNow
- FireHydrant
- Opsgenie

Integrations (2):
- SonarQube
- Synk
- Wiz

Integrations (3):
- Kubecost
- Opencost

Integrations (4):
- Kafka
- ArgoCD

## Updated docs pages

Please also include the path for the updated docs
Integrations (1):
- docs/build-your-software-catalog/sync-data-to-catalog/incident-management/firehydrant.md
- docs/build-your-software-catalog/sync-data-to-catalog/incident-management/opsgenie.md
- docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty.md
- docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md

Integrations (2):
- docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/snyk/snyk.md
- docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/sonarqube/sonarqube.md
- docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md

Integrations (3):
- docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md
- docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/opencost.md

Integrations (4):
- docs/build-your-software-catalog/sync-data-to-catalog/event-processing/kafka.md
- docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md [hunk 1]
```diff
@@ -264,6 +264,66 @@ pipeline {
...
+| `OCEAN__INITIALIZE_PORT_RESOURCES`       | Default true, When set to false the integration will not create default blueprints and the port App config Mapping | ❌       |
+| `OCEAN__INTEGRATION__IDENTIFIER`         | Change the identifier to describe your integration, if not set will use the default one                            | ❌       |
+| `OCEAN__PORT__CLIENT_ID`                 | Your port client id                                                                                                | ✅       |
+| `OCEAN__PORT__CLIENT_SECRET`             | Your port client secret                                                                                            | ✅       |
+| `OCEAN__PORT__BASE_URL`                  | Your port base url, relevant only if not using the default port app                                                | ❌       |
+
+<br/>
+
+
+Here is an example for `argocd-integration.yml` pipeline file:
```
Comment (yairsimantov20) [inline docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md:290]: shouldn't the file name be `gitlab-ci.yml`

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md [hunk 2]
```diff
@@ -282,7 +282,56 @@ steps:
...
+:::
+
+Make sure to [configure the following GitLab variables](https://docs.gitlab.com/ee/ci/variables/#for-a-project):
+
+<DockerParameters />
+
+<br/>
 
+
+Here is an example for `kubecost-integration.yml` pipeline file:
```
Comment (yairsimantov20) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md:299]: shouldn't it be gitlab-ci.yml?

Comment (yairsimantov20) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md:299]: applies to all files

File: docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md [hunk 3]
```diff
@@ -264,6 +264,66 @@ pipeline {
...
+| `OCEAN__INTEGRATION__CONFIG__SERVER_URL` | The ArgoCD server URL                                                                                              | ✅       |
+| `OCEAN__INITIALIZE_PORT_RESOURCES`       | Default true, When set to false the integration will not create default blueprints and the port App config Mapping | ❌       |
+| `OCEAN__INTEGRATION__IDENTIFIER`         | Change the identifier to describe your integration, if not set will use the default one                            | ❌       |
+| `OCEAN__PORT__CLIENT_ID`                 | Your port client id                                                                                                | ✅       |
+| `OCEAN__PORT__CLIENT_SECRET`             | Your port client secret                                                                                            | ✅       |
+| `OCEAN__PORT__BASE_URL`                  | Your port base url, relevant only if not using the default port app                                                | ❌       |
```
Comment (yairsimantov20) [inline docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md:285]: can you remove port base url from all examples?

**Review by yairsimantov20 — DISMISSED** [review]
some comments

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md (hunk 2 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-cost/kubecost.md]: resolved

File: docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md (hunk 1 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md]: yes, it should be. thank you for picking this up

File: docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md (hunk 3 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md]: removed it from all integrations in this PR. I will remove it also from [this ](https://github.com/port-labs/port-docs/pull/1234)

Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md]: Should I also be removing it from the helm parameters ie. 

	`--set port.baseUrl="https://api.getport.io"  \`
	
I noticed some integrations have it and others don't

Comment (MPTG94) [reply docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md]: Yes let's also remove it from the helm params

Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/argocd/argocd.md]: Hi. I have removed this now

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**